api/routes.js

```diff
@@ -308,4 +308,8 @@ exports.set = function(app, mongo, log) {
     app.get(URL_TEAM_PICK, (request, response) => {
         response.send(JSON.stringify(teamsJSON, null, 2));
     });
+
+    app.get('/api/healthcheck', (request, response) => {
+        response.send({ status : 'OK' });
```

Comment:
you could also make a request to database before sending OK, some "find" command is enough to make sure everything is working